### PR TITLE
feat(profiling): Add go to transaction button

### PR DESCRIPTION
**Pull request by Zylphrex**
In similar fashion to the go to profile button on a transaction details page, this introduces a go to transaction button on the profile details page.

# Screenshot
image

### CODE REVIEW

File: static/app/components/profiling/profileHeader.tsx [hunk 1]
```diff
@@ -49,6 +69,13 @@ function ProfileHeader() {
           ]}
         />
       </Layout.HeaderContent>
+      <Layout.HeaderActions>
+        {transactionTarget && (
+          <Button onClick={handleGoToTransaction} to={transactionTarget}>
```
Comment (JonasBa) [inline static/app/components/profiling/profileHeader.tsx:74]: @Zylphrex we also have the transaction in the details drawer - we should make sure we add it there too. Also, any chance we can make this an actual link vs a button and get some semantics html points :D?

**Comment by Jesse-Box** [comment]
Design Nit, Vertical alignment of Breadcrumbs 👇 
image

File: static/app/components/profiling/profileHeader.tsx (hunk 1 above)
Comment (Zylphrex) [reply static/app/components/profiling/profileHeader.tsx:74]: Yeah let's add it to the details draw too.

This button is an anchor under the hood right now, is that what you mean?

**Review by JonasBa — APPROVED** [review]
Send it

**Pull request merged**